cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/VirtualOutletEndpointInput.java

```diff
@@ -72,21 +72,22 @@ private GetOne(String officeId, String projectId, String outletName) {
             this.projectId = Objects.requireNonNull(projectId, "Project Id required for getOne outlet endpoint");
         }
 
-        public String getOutletName() {
+        public String outletName() {
             return outletName;
         }
 
-        public String getOfficeId() {
+        public String officeId() {
             return officeId;
         }
 
-        public String getProjectId() {
+        public String projectId() {
             return projectId;
         }
 
         @Override
         protected HttpRequestBuilder addInputParameters(HttpRequestBuilder httpRequestBuilder) {
-            return httpRequestBuilder.addQueryHeader(ACCEPT_QUERY_HEADER, ACCEPT_HEADER_V1);
+            return httpRequestBuilder.addQueryParameter(OFFICE_QUERY_PARAMETER, officeId)
```

Comment:
Why is the officeId added as a query parameter when it's part of the path?  what does the CDA controller have for these parameters?

Reply:
Removed extra query parameter to match the CDA controller